CMakeLists.txt

```diff
@@ -31,5 +31,20 @@ if(NOT IS_LOOKUP_NODE)
     endif()
 endif()
 
+if(NOT WIN32)
+    SET( LevelDB_LIBRARIES leveldb )
+endif()
+if(WIN32)
+    FIND_LIBRARY( LevelDB_LIBRARY libleveldb )
+    SET( LevelDB_LIBRARIES ${LevelDB_LIBRARY} shlwapi )
+endif()
+
+# Dependencies
+FIND_PACKAGE(Boost 1.33 REQUIRED COMPONENTS filesystem system unit_test_framework)
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIRS})
```

Comment:
❓ Do we still need to specify `-lboost_system -lboost_filesystem` in CMAKE_EXE_LINKER_FLAGS above? It seems like this is all covered in the individual projects.

Reply:
➡️ Now that Travis-CI is set up, I was able to remove these lines and see that the build still works without them.